### PR TITLE
Agent: fix `pnpm generate-agent-kotlin-bindings`

### PR DESCRIPTION
**Pull request by olafurpg**
Previously, this command only ran the bindings generator, not the scip-typescript indexing step. Now, this command runs everything.

## Test plan

Green CI

### CODE REVIEW

File: agent/bindings/kotlin/lib/src/main/kotlin/com/sourcegraph/cody/protocol_generated/ContextFilters.kt
Comment (odisseus) [inline agent/bindings/kotlin/lib/src/main/kotlin/com/sourcegraph/cody/protocol_generated/ContextFilters.kt:1]: Why empty file?

File: agent/bindings/kotlin/lib/src/main/kotlin/com/sourcegraph/cody/protocol_generated/WebviewRecordEventParameters.kt
Comment (odisseus) [inline agent/bindings/kotlin/lib/src/main/kotlin/com/sourcegraph/cody/protocol_generated/WebviewRecordEventParameters.kt:1]: Why empty file?

File: agent/src/index.test.ts [hunk 1]
```diff
@@ -97,7 +97,7 @@ describe('Agent', () => {
         })
         // TODO(dpc): Integrate file-based .cody/ignore with ignore/test
         expect(ignore.policy).toBe('use')
-    }, 10_000)
+    }, 20_000)
```
Comment (odisseus) [inline agent/src/index.test.ts:100]: Why?

**Review by odisseus — APPROVED** [review]
Wow, that's a lot of changed files. But I guess you cannot break anything if you don't add any code?

**Comment by olafurpg** [comment]
@odisseus My plan was not to remove all the files, they got deleted since I ran the generation script and it's erroring. Not a problem to merge though, I will open a new PR adding everything back (and re-enabling the CI check)

**Comment by olafurpg** [comment]
@odisseus there's additional context you may be missing. In JVM codebases, it's conventional to not check in files but it's more common in other ecosystems to do this. For this specific situation, it's convenient to have the files checked for a variety of reasons. The biggest reason is that the script to generate the bindings has some complicated dependencies that I wanted to avoid pulling into the JB build, it's simpler to just copy-paste generated files from the git tree.

**Comment by dominiccooney** [comment]
My $0.02, +1 to checking in the generated output for the reasons @olafurpg mentions. It makes it easier to review the generated output, too, if we change the generation.

**Pull request merged**